CONTENTS.md

```diff
@@ -0,0 +1,420 @@
+* Usage: ?
+
+#### totest-manager
+
+Releases distribution snapshots to openQA and publishes if the result is positive.
+
+* Sources: [totest-manager.py](totest-manager.py) and [ttm](ttm)
+* Documentation: [ttm/README.md](ttm/README.md)
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### unmaintained
+
+Finds unmaintained binaries sourced from SLE.
+
+* Sources: [unmaintained.py](unmaintained.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### sync-rebuild (obsolete)
+
+Syncs openSUSE:Factory and openSUSE:Factory:Rebuild. This feature was already
+merged into the [accept
+command](https://github.com/openSUSE/openSUSE-release-tools/commit/87c891662015f14421c2315210c248e712e697c8)
+of the staging projects plug-in.
+
+* Sources: [sync-rebuild.py](sync-rebuild.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### bugowner
+
+Manages bugowner information
+
+* Sources: [bugowner.py](bugowner.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### bs_mirrorfull
+
+Mirrors repositories from the build service to a local directory.
+
+* Souces: [bs_mirrorfull](bs_mirrorfull)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: Used by other tools like `pkglistgen` or `repocheck`
+
+#### biarchtool
+
+Manages biarch packages
+
+* Source: [biarchtool.py](biarchtool.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### build-fail-reminder
+
+Sends e-mails about packages failing to build for a long time.
+
+* Sources: [build-fail-reminder.py](build-fail-reminder.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: gocd
+
+#### checknewer
+
+Checks if all packages in a repository are newer than all other repositories.
+
+* Sources: [checknewer.py](checknewer.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### deptool
+
+Assists in debugging dependencies
+
+* Sources: [deptool.py](deptool.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### requestfinder
+
+Allows to retrieve requests from OBS with quite elaborated queries.
+
+* Sources: [requestfinder.py](requestfinder.py)
+* Documentation: --
+* Package: openSUSE-release-tools
+* Usage: ?
+
+#### update_crawler (obsoleted by [origin-manager](#origin-manager))
+
+Create SRs for Leap.
+
+* Sources: [update_crawler.py](update_crawler.py) and [script](script).
+* Documentation: --
+* Package: openSUSE-release-tools-leaper
+* Usage: see [leaper](#leaper)
+
+#### create_staging
+
+Scripts and templates to create staging projects.
+
+* Sources: [staging_templates](staging_templates)
+* Documentation: --
+* Package: --
+* Usage: ?
+
+### Bots
+
+#### check_maintenance_incidents
+
+Handles maintenance incident requests
+
+* Sources: [check_maintenance_incidents.py](check_maintenance_incidents.py)
+* Documentation: [docs/maintbot.asciidoc](docs/maintbot.asciidoc)
+* Package: openSUSE-release-tools-maintenance
+* Usage: gocd
```

Comment:
[This comment](https://github.com/openSUSE/openSUSE-release-tools/issues/1662#issuecomment-512953010) seems to suggest this could/should/was replaced by origin-manager. We need to investigate further. Maybe it's another case of obsolete tool we can shutdown.

Reply:
seems very outdated script